components/gitpod-protocol/src/protocol.ts

```diff
@@ -211,6 +211,12 @@ export interface AdditionalUserData {
     dotfileRepo?: string;
     // preferred workspace classes
     workspaceClasses?: WorkspaceClasses;
+    // whether running prebuilds should be ignored on start
+    ignoreRunnningPrebuilds?: boolean;
```

Comment:
Would it make sense to encapsulate these options in a sub-shape ala `workspaceStartOptions` or so? That would make some of the other code shorter as well, and help keeping this one clean(er).